### ISSUE TITLE
Can we get only circular bitmap only?

### ISSUE THREAD

**Issue by droidstered** [adamstyrc/cookie-cutter#10]
I know there is different crop shape option.
But my question is that can we crop image as a circle as you have implemented in this project and get rectangular result bitmap instead of circular bitmap?

I hope you will understand my question.

**Comment by adamstyrc** [comment]
All bitmaps are rectangular. The area outside the circle will be
transparent.

Does it answers your question?

14.02.2018 1:48 PM "Ravi Vachhani" <notifications@github.com> napisał(a):

**Comment by droidstered** [comment]
No. This is not the answer of my question. I was not asking of cropping screen. I was telling about the result bitmap.
